### DISCUSSION TITLE
Width of columns and rows in a complex table in EUSurvey

### DISCUSSION THREAD

**Discussion by taisymac** [EUSurvey/EUSURVEY#559] [General]
Hello, can I narrow the width of columns and rows in a complex table in EUSurvey

**Comment by SJ41400** [comment]
Un peu tard, mais...
Par des CSS; NON, elles sont cadenassées sur un nombre limité d'objets et d'attributs, 
Du code: il est supprimé, donc aucun objet, autres que ceux éditables, n'est accessible... ex: col  
Donc juste bidouiller dans "code source" comme par exemple dans une colonne: <div class="mycenter" **style="width: 400px;"**>Titre B<?/div>  et ajuster les autres... je ne suis pas aller plus loin bonjour les dégats!!!
**### A priori on ne pas visualiser sur différents appareils en dév. .... 
ça donne quoi comme résultat????????????????????????**
